src/Page/Partners/Partner_.elm

```diff
@@ -286,6 +275,52 @@ viewInfo localModel { partner, events } =
         ]
 
 
+viewPartnerEvents : Model -> Data -> Html Msg
+viewPartnerEvents localModel { partner, events } =
+    let
+        eventAreaTitle =
+            h3 [ css [ smallInlineTitleStyle, color white ] ] [ text (t (PartnerUpcomingEventsText partner.name)) ]
+    in
+    section [ id "events" ]
+        (if List.length events > 0 then
```

Comment:
It feels a little nested - logic if / else-y and not that easy to follow.

I think there is a simplification we could do here, but if you don't have time / inclination to refactor, I'm happy to accept the tech debt at this point.